docs/articles/documentation/reference/test-api/test/page.md

```diff
@@ -31,14 +31,24 @@ test
 
 If the start page is not specified, the default URL is `about:blank`.
 
-You can use the `file://` scheme or relative paths to test web pages in local directories.
+You can use the `file://hostname/path` scheme or relative paths to test web pages in local and network directories.
```

Comment:
It's better to add link to [specification](https://tools.ietf.org/html/rfc8089) or [Wikipedia](https://en.wikipedia.org/wiki/File_URI_scheme). I prefer the Wikipedia link because the Wikipedia's description is simpler.

You can use the [file URI scheme](https://en.wikipedia.org/wiki/File_URI_scheme) or ...

Reply:
```suggestion
You can use the `file://hostname/path` [URI scheme](https://en.wikipedia.org/wiki/File_URI_scheme) or relative paths to test web pages in local and network directories.
```